docs/source/hardware/hardware_overview.rst

```diff
@@ -3,129 +3,73 @@
 ******************
 Hardware overview
 ******************
-The Discovery cluster provides you with access to over 24,000 CPU cores and over 200 GPUs. Discovery is connected
+NEU's HPC provides you with access to over 1024 CPU nodes, 50,000 CPU cores, and over 200 GPUs and is connected
 to the university network over 10 Gbps Ethernet (GbE) for high-speed data transfer.
 Compute nodes are connected to each other with either 10 GbE or a high-performance HDR200 InfiniBand (IB) interconnect
 running at 200 Gbps (with some nodes running HDR100 IB, if HDR200 IB is not supported on those nodes).
 
-CPU nodes and their feature names
+CPU nodes
 =================================
-As of February 2021, all previous CPU node feature names were updated to human-friendly feature names following the
-archspec microarchitecture specification (https://archspec.readthedocs.io/en/latest/index.html). **This update is only to the CPU node feature names.**
-GPU node feature names remain unchanged at this time. Table 1 shows the new feature names and
-their corresponding previous feature names. Note that the old feature names will be removed from use with your jobs as of March 3, 2021.
-Make sure to update your scripts with the new feature names if you had used any of the old feature names previously.
 
-**Table 1: CPU Nodes**
+Table 1 below shows the feature names, number of nodes by partition type (public, private, and total), and the RAM memory range per node. The feature name follows archspec microachitechture specification (https://archspec.readthedocs.io/en/latest/index.html). 
```

Comment:
Change number of nodes by partition type (public, private, and total) 
to
number of nodes in public & private partitions. 

I think we can take out "total" from the table as people with access to only public partitions won't have access to private ones, so total won't matter.

Reply:
Change specification (https://archspec.readthedocs.io/en/latest/index.html)
to
(`specification <https://archspec.readthedocs.io/en/latest/index.html>`_)